tests/common

```diff
@@ -0,0 +1,46 @@
+#!/bin/ksh
+set -e 
+failed=0
+
+workdir="$PWD"
+tmpdir=$(mktemp -d -p $workdir)
+
+port=13245
+www=$(mktemp -d -p $tmpdir)
+shellweb_cmd="${workdir}/shellweb -d $www -p $port"
+
+test_name=${0##*/test-}
+
+run_shellweb(){
+	$shellweb_cmd &
+	sleep 0.1
+	
+}
+
+kill_shellweb() {
+	pkill -u $(id -u) -f "$shellweb_cmd"
+	pkill -u $(id -u) -f nc
```

Comment:
"pkill -f nc" looks like too much: it will kill "encrypt" process, for example. Better use smth. like "pkill -x nc" instead here.